views/error.ejs

```diff
@@ -1,10 +1,9 @@
 <% layout('layouts/boilerplate')%>
-<div class="row">
-  <div class="col-6 offset-3">
-    <div class="alert alert-danger" role="alert">
-      <h4 class="alert-heading"><%=err.message%></h4>
+<div class="card">
+  <div class="card-box">
+    <div class="field-box">
```

Comment:
`field-box` - seems that it was meant for the form elements? Besides there is not a class with the name of `field-box`.

Reply:
field-box'es are no more